### PR TITLE
Updating the Running Cloud Functional Tests and adding the Running Non-cloud Functional Tests docs

### PR DESCRIPTION
**Pull request by ytimocin**
# Description

Updating the Running Cloud Functional Tests and adding the Running Non-cloud Functional Tests docs

## Type of change
- This pull request is a minor refactor, code cleanup, test improvement, or other maintenance task and doesn't change the functionality of Radius (issue link optional).
Fixes: #issue_number

## Contributor checklist
Please verify that the PR meets the following requirements, where applicable:

- [ ] An overview of proposed schema changes is included in a linked GitHub issue.
- [ ] A design document PR is created in the [design-notes repository](https://github.com/radius-project/design-notes/), if new APIs are being introduced.
- [ ] If applicable, design document has been reviewed and approved by Radius maintainers/approvers.
- [ ] A PR for the [samples repository](https://github.com/radius-project/samples) is created, if existing samples are affected by the changes in this PR.
- [ ] A PR for the [documentation repository](https://github.com/radius-project/docs) is created, if the changes in this PR affect the documentation or any user facing updates are made.
- [ ] A PR for the [recipes repository](https://github.com/radius-project/recipes) is created, if existing recipes are affected by the changes in this PR.

### CODE REVIEW

**Review by kachawla — COMMENTED** [review]
Thanks for updating this. A few suggestions.

**Review by superbeeny — COMMENTED** [review]
Looks good, something that I just went through was referencing how to run [the control plane locally](https://github.com/radius-project/radius/blob/main/docs/contributing/contributing-code/contributing-code-control-plane/running-controlplane-locally.md) and then running the functional tests.

my 2c

**Comment by ytimocin** [comment]
Closing this PR in favor of this one: https://github.com/radius-project/radius/pull/8087.

File: docs/contributing/contributing-code/contributing-code-tests/running-noncloud-functional-tests.md
[the remark this answers is not in the document]
Comment (ytimocin) [reply docs/contributing/contributing-code/contributing-code-tests/running-noncloud-functional-tests.md:1]: @kachawla I captured the information in one file in the other PR: #8087.

File: test/execute_cloud_functional_tests.sh [hunk 1]
```diff
@@ -31,7 +33,5 @@ resp=$(az deployment group create --resource-group $resourcegroup --template-fil
 cat resp
 
 export AZURE_COSMOS_MONGODB_ACCOUNT_ID=$(jq -r '.properties.outputs.cosmosMongoAccountID.value' <<<"${resp}")
-make test-functional-corerp
-make test-functional-msgrp
```
[the remark this answers is not in the document]
Comment (ytimocin) [reply test/execute_cloud_functional_tests.sh:35]: Yes.

File: test/execute_cloud_functional_tests.sh [hunk 2]
```diff
@@ -31,7 +33,5 @@ resp=$(az deployment group create --resource-group $resourcegroup --template-fil
...
-make test-functional-daprrp
-make test-functional-datastoresrp
+# We probably need AWS_ACCOUNT_ID and AWS_REGION as well
```
[the remark this answers is not in the document]
Comment (ytimocin) [reply test/execute_cloud_functional_tests.sh:36]: This is also deleted in the other one.

File: test/execute_cloud_functional_tests.sh
[the remark this answers is not in the document]
Comment (ytimocin) [reply test/execute_cloud_functional_tests.sh:1]: Answered in the other PR.

**Pull request closed without merging**